emukit/examples/profet/meta_benchmarks/meta_fcnet.py

```diff
@@ -0,0 +1,90 @@
+import pickle
+import torch
+import numpy as np
+
+from functools import partial
+from typing import Tuple
+
+from emukit.core import ContinuousParameter, ParameterSpace, DiscreteParameter
+from emukit.core.loop.user_function import UserFunctionWrapper
+from emukit.examples.profet.meta_benchmarks.architecture import get_default_architecture_classification, \
+    get_default_architecture_cost
+
+
+def meta_fcnet(fname_objective: str, fname_cost: str, noise: bool=True) -> Tuple[UserFunctionWrapper, ParameterSpace]:
+    """
+    Interface to the Meta-FCNet benchmark which imitates the hyperparameter optimization of a
+    fully connected neural network on OpenML like classification datasets.
+    Offline generated function samples can be download here:
+
+    http://www.ml4aad.org/wp-content/uploads/2019/05/profet_data.tar.gz
+
+    NOTE: make sure that the index for the objective function and the cost function match,
+    e.g for sample_objective_i.pkl  and sample_cost.pkl the index i should be the same.
```

Comment:
should this say sample_cost_i?
```suggestion
    e.g for sample_objective_i.pkl  and sample_cost_i.pkl the index i should be the same.
```

Reply:
yes indeed